src/components/PopupWithForm.js

```diff
@@ -3,7 +3,22 @@ export default class PopupWithForm extends Popup {
   constructor(popupSelector, handleFormSubmit) {
     super({ popupSelector });
     this._handleFormSubmit = handleFormSubmit;
-    this._popupForm = this._popupElement.querySelector(".modal__form");
+
+    // this._popupDeleteForm =
+    //   this._popupElement.querySelector("#delete-modal-form");
+    // this._popupButton = this._popupElement.querySelector("#delete-sure-button");
+
+    // this._pictureChangeButton = this._popupElement.querySelector(
+    //   "#profile-change-button"
+    // );
+    // this._pictureChangeModal = this._popupElement.querySelector(
+    //   "#picture-modal-form"
+    // );
+    // this._pictureChangeSubmit = this._popupElement.querySelector(
+    //   "#picture-save-button"
+    // );
+    // this._modalButton = this._popupElement.querySelector(".modal__button");
+    // this._popupForm = this._popupElement.querySelector(".modal__form");
```

Comment:
**Could be improved**

Please, find the button, form, inputs only once in the constructor

This was the right variant